src/fsharp/ConstraintSolver.fs

```diff
@@ -1063,153 +1066,176 @@ and SolveMemberConstraint (csenv:ConstraintSolverEnv) ignoreUnresolvedOverload p
                    checkRuleAppliesInPreferenceToMethods argty2 argty1) ->
                    
           match GetMeasureOfType g argty1 with
-          | Some (tcref, ms1) ->
+          | Some (tcref, ms1) -> trackErrors {
```

Comment:
I think I expect the `trackErrors {` to be around the entire

```
    match minfos, tys, memFlags.IsInstance, nm, argtys with
```
rather than each branch of that